pages/common/git-verify-commit.md

```diff
@@ -0,0 +1,17 @@
+# git verify-commit
+
+> Check for GPG verification of commits.
+> If no commits are verified, nothing will be printed, regardless of options specified.
+> More information: <https://git-scm.com/docs/git-verify-commit>.
+
+- Check commits for a GPG signature:
+
+`git verify-commit {{commit_hash}} {{optional_commit_hash(s')}}`
```

Comment:
Add suggestions here

Reply:
I like `{{commit_hash1 commit_hash2 ...}}`.  I don't think it indicates much that 2+ commits are required, and we do it in quite a few other places.

Reply:
But, at least, `optional_commit_hash(s')` is wrong — it should be `optional_commit_hash(es)`, at least.

Reply:
```suggestion
`git verify-commit {{commit_hash1 commit_hash2 ...}}`
```

Reply:
What it is 

`git verify-commit {{commit_hash1 optional_commit_hash2 ...}}`

I think the ... is needed now, however as a user I would feel you need at least 2 to use the command

Reply:
I like that 👍🏻

Reply:
should all of them have it? or just the top command?

Reply:
Oh, I didn't see this.  There are few enough commands so that it's fine if they all have it, I think.

Reply:
Resolving, no one said anything about it when the wrong commands (irrelevant) were in all of the commands